src/quick-view.scss

```diff
@@ -0,0 +1,100 @@
+@import "./new-settings";
+@import "./mixins";
+
+$block: #{$fd-namespace}-quick-view;
+
+$fd-quick-view-avatar-indent: 0.75rem;
+$fd-quick-view-title-bottom-indent: 0.5rem;
+
+.#{$block} {
+  @include fd-reset();
+
+  min-width: 20rem;
+  max-width: 30rem;
```

Comment:
Where are those limits from?

Reply:
I wasn't able to find any info about those limits in the design docs,
but this component has already pre-defined content and I think those limits are necessary for the view perception.

For `min-width` I used a width value from [examples], (https://sapui5.hana.ondemand.com/#/entity/sap.m.QuickView/sample/sap.m.sample.QuickView) They all have `320px` width.

I wasn't able to use max-width as well from those examples because it is dynamic there,
so `30rem` is my assumption

Reply:
Could you then put it as a hardcoded value into examples? You can use inline-styling or check how it's done with dialog